source/speech/speech.py

```diff
@@ -1111,7 +1113,6 @@ def speak(  # noqa: C901
 	_speechState.beenCanceled = False
 	# Filter out redundant LangChangeCommand objects
 	# And also fill in default values
-	autoLanguageSwitching = config.conf["speech"]["autoLanguageSwitching"]
```

Comment:
can you please make sure to replace all usages of this with `shouldGetLanguageForTextInfo` rather than just removing `autoLanguageSwitching`?

Reply:
you might want to remove the comments above this